doc/stdlib/make-library-index

```diff
@@ -36,7 +36,8 @@ for k in $LIBDIRS; do
                 fi
 	    else
 	        if [ $h = 0 ]; then
-		    echo Warning: $k/$b.v will be hidden from the index
+		    # Skipping file from the index
+                    :
```

Comment:
What's this `:` for?

Why not just invert the condition?

Reply:
`:` is the bash operator for a no-op

It seemed to me that the logic was clearer that way: explicitly skip the file from the index.